ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -85,6 +85,8 @@ conformance_test(
             "CommandDeduplicationIT:SimpleDeduplicationBasic",  # sync vs async error (part of canton #6301)
             "CommandDeduplicationIT:DeduplicateSubmitterBasic",
             "CommandDeduplicationIT:DeduplicateUsingOffsets",  # temporary disabled until we adapt it for canton
+            "MultiPartySubmissionIT:MPSLookupOtherByKeyInvisible",  # Enable once error parsing more permissive
+            "MultiPartySubmissionIT:MPSCreateInsufficientAuthorization",  # canton returns INTERNAL instead of INVALID_ARGUMENT
```

Comment:
Is that expected? Is the canton team aware of this?

Reply:
No idea but I just moved them upwards.

@oliverse-da, are you aware we're ignoring these test cases?

Reply:
Was aware of the first one failing, but not the second one. 👀

Reply:
Both pass on my machine so I'm enabling them both. If they pass on CI, win.